src/main.cpp

```diff
@@ -71,7 +71,7 @@ int main (int argc, char *argv[])
             std::unique_ptr<PHiLiP::Tests::TestsBase> test = PHiLiP::Tests::TestsFactory<max_dim,max_nstate>::create_test(&all_parameters,parameter_handler);
             run_error = test->run_test();
             pcout << "Finished integration test with run error code: " << run_error << std::endl;
-        }
+    }
```

Comment:
revert all unnecessary changes like this; please do another self-review on the PR